src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,61 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
+
+  isAlive() {
+    return this.health > 0;
+  }
+
+  die() {
+    const index = Animal.alive.indexOf(this);
+
+    if (index > -1) {
+      Animal.alive.splice(index, 1);
```

Comment:
To improve performance and align with best practices, refactor this to use the `filter` method instead of `indexOf` and `splice` to remove an animal from the `Animal.alive` array.